utils/cards.ts

```diff
@@ -0,0 +1,26 @@
+import * as childProcess from "child_process";
+import * as util from "util";
+
+const isWin = process.platform.includes("win") && process.platform !== "darwin";
```

Comment:
As I found at types  description
```   
type Platform = 'aix'
        | 'android'
        | 'darwin'
        | 'freebsd'
        | 'linux'
        | 'openbsd'
        | 'sunos'
        | 'win32'
        | 'cygwin';
```
I understand that it's very small likelyhood of mistake but if we check `darwin` maybe we should also check `cygwin`?

Reply:
replaced with `process.platform.startsWith('win')`